scio-core/src/main/scala/com/spotify/scio/testing/TestDataManager.scala

```diff
@@ -171,6 +182,9 @@ private[scio] object TestDataManager {
     require(closed(testId), "ScioContext was not executed. Did you forget .run()?")
     closed -= testId
   }
+
+  def overrideTransforms(testId: String, pipeline: Pipeline): Unit =
+    transformOverrides.get(testId).foreach(_.overrideTransforms(pipeline))
```

Comment:
```suggestion
    transformOverrides.get(testId).foreach(overrides => pipeline.replaceAll(overrides.toList.asJava))
```

Reply:
This is equivalent and I'm all for nuking random classes but it'd also break the pattern used by the other parts of `TestDataManager`.

Reply:
eh, I think it's ok to break pattern and get rid of the `TestTransform` class, since it doesn't have much functionality outside of holding the Set of PTransform overrides.

Reply:
Done.